packages/react-ui/src/theme.js

```diff
@@ -1,100 +1,98 @@
-export const animation = {
-  timings: {
-    transition: "250ms"
-  }
-};
+export const variants = [
```

Comment:
can you preserve the initial order to ease the review, it's hard to track what modification you have done